readmes/INSTALL_ANDROID.md

```diff
@@ -0,0 +1,157 @@
+   project(':gvr_common').projectDir = new File('../node_modules/@viro-community/react-viro/android/gvr_common')
+   project(':viro_renderer').projectDir = new File('../node_modules/@viro-community/react-viro/android/viro_renderer')
+   project(':react_viro').projectDir = new File('../node_modules/@viro-community/react-viro/android/react_viro')
+   ```
+
+   See an [example](https://github.com/ViroCommunity/starter-kit/blob/master/android/settings.gradle) here.
+
+4. In your `android/gradle/wrapper/gradle-wrapper.properties` set the `distributionUrl` to at least 6.5:
+
+   ```properties
+   distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-bin.zip
+   ```
+
+   See an [example](https://github.com/ViroCommunity/starter-kit/blob/master/android/gradle/wrapper/gradle-wrapper.properties) here.
+
+5. Now add the Viro package to your `MainApplication.java`:
+
+   - Add the following line to the end of the import list:
+
+     ```java
+     import com.viromedia.bridge.ReactViroPackage;
+     ```
+
+   - Add the imported package inside the `getPackages` method:
+
+     ```java
+     @Override
+     protected List<ReactPackage> getPackages() {
+       @SuppressWarnings("UnnecessaryLocalVariable")
+       List<ReactPackage> packages = new PackageList(this).getPackages();
+
+       // Add this line
+       packages.add(new ReactViroPackage(ReactViroPackage.ViroPlatform.valueOf("AR")));
+
+       return packages;
+     }
+
+     ```
+
+     You can replace the string `AR` with one of the following depending on your needs: `GVR`, `OVR_MOBILE`, `AR`.
+
+   See an [example](https://github.com/ViroCommunity/starter-kit/blob/master/android/app/src/main/java/com/myviroapp/MainApplication.java) here.
+
+6. In your `android/app/src/main/AndroidManifest.xml`:
+
+   **For AR**
+
+   - Add the following line to the `<application>` node, this enables ARCore:
+
+     ```xml
+     <meta-data android:name="com.google.ar.core" android:value="optional" />
+     ```
+
+     _Note_: If you want to restrict your app to ARCore-only devices, set the `android:value` to `"required"`
+
+   - Add the camera permission to the `<manifest>` node below other permissions:
+
+     ```xml
+     <uses-permission android:name="android.permission.CAMERA" />
+     ```
+
+     There are quite a few other camera-related permissions and features you might also want to specify, depending on your app:
+
+     ```xml
+     <!-- You may need these if doing any screen recording from within the app -->
+     <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"/>
+     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
+     <!-- Other camera related features -->
+     <uses-feature android:name="android.hardware.camera" />
+     <uses-feature android:name="android.hardware.camera.autofocus" android:required="false" tools:replace="required"/>
```

Comment:
It's not written anywhere here that the `tools:` namespace must be included. See https://github.com/ViroCommunity/starter-kit/pull/3.